src/Webhooks/Webhooks.csproj

```diff
@@ -1,311 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
?

Reply:
I deleted csproj, Louise told me to do decommit, idk if that is he was refering at 👉🏻👈🏻

Reply:
> I deleted csproj

Why